backend/routes.go

```diff
@@ -78,21 +81,33 @@ func (s *server) handleAddUser() http.HandlerFunc {
 //GET: endpoint for all articles
 func (s *server) handleGetArticles() http.HandlerFunc {
 	return func(w http.ResponseWriter, req *http.Request) {
+
+
 		type ArticleJson struct {
 			Title string `json:"title"`
 			URL   string `json:"url"`
 			//Votes []Vote `json:"votes"`
 		}
 
-		results, err := s.db.GetAllArticles()
+		v := req.URL.Query()
+		//DEFAULT FLAG = -1
+		var numArticles = -1
 
+		//No count value? RETURN ALL
+		if v.Get("count") != "" {
+			numArticles, _ = strconv.Atoi(v.Get("count"))
```

Comment:
Lets not discard the error. It could be a valid issue with the caller sending an invalid count.

We should return an error to the caller in this case with `http.StatusBadRequest`